demo.py

```diff
@@ -173,16 +173,16 @@ def build_dqm(W, C, n, p, a, verbose=True):
     gamma1 = 150
     for i in range(n):
         for j in range(n):
-            dqm.set_linear_case(i,j, dqm.get_linear_case(i,j) + 1*gamma1)
             if i != j:
+                dqm.set_linear_case(i,j, dqm.get_linear_case(i,j) + 1*gamma1)
```

Comment:
just a nitpick: would be good for all the spacing for function args to be consistent